pasay-telegram-bot/pasay_bot/state/store.py

```
@@ -558,60 +519,519 @@ def get_idempotency(self, key: str) -> Optional[dict]:
+            cur = self._cursor()
+            try:
+                cur.execute(
+                    "INSERT INTO bs_idempotency_keys "
+                    "(key, kind, resource, status, result_json, created_at, expires_at) "
+                    "VALUES (%s, %s, %s, %s, NULL, %s, %s) "
+                    "ON CONFLICT (key) DO NOTHING",
+                    (key, kind, resource, status, now, expires),
+                )
+                self._conn.commit()
+                return (cur.rowcount or 0) > 0
+            except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)